docs/configuration/configuration.md

```diff
@@ -285,6 +285,10 @@ marathon_sd_configs:
 nerve_sd_configs:
   [ - <nerve_sd_config> ... ]
 
+# List of Nomad service discovery configurations.
+nomad_sd_configs:
```

Comment:
This has to be listed twice in the configuration: once in the alerting and one in the scraping configs.